plugins/autofarm.cpp

```diff
@@ -213,11 +203,11 @@ class AutoFarm {
             farm->plant_id[season] = new_plant_id;
             out << "autofarm: changing farm #" << farm->id <<
                 " from " << get_plant_name(old_plant_id) <<
-                " to " << get_plant_name(new_plant_id) << endl;
+                " to " << get_plant_name(new_plant_id) << '\n';
```

Comment:
I think it's functionally equivalent, but isn't the standard to use `std::endl` in this context?

Reply:
`endl` forces an output flush, there's no reason to do that here